### PR TITLE
Copter: Change messages

### PR DESCRIPTION
**Pull request by muramura**
I think it's better to format it with other messages.

### CODE REVIEW

File: ArduCopter/AP_Arming.cpp [hunk 1]
```diff
@@ -260,7 +260,7 @@ bool AP_Arming_Copter::parameter_checks(bool display_failure)
                 uint16_t terr_pending, terr_loaded;
                 copter.terrain.get_statistics(terr_pending, terr_loaded);
                 if (terr_pending != 0) {
-                    check_failed(ARMING_CHECK_PARAMETERS, display_failure, "RTL_ALT_TYPE=1, waiting for terrain data");
```
Comment (rmackay9) [inline ArduCopter/AP_Arming.cpp]: we should leave this one as it was..

Otherwise the change looks good, thanks!

**Comment by khancyr** [comment]
@muramura could you update with the requested change ?

**Comment by khancyr** [comment]
@rmackay9 I think the comma was removed to make the message looks like the other from the same kind. In fact, I prefer with the comma too, so I put the comma on each !
Should be good to go now

File: ArduCopter/AP_Arming.cpp [hunk 2]
```diff
@@ -237,24 +237,24 @@ bool AP_Arming_Copter::parameter_checks(bool display_failure)
             // get terrain source from wpnav
             switch (copter.wp_nav->get_terrain_source()) {
             case AC_WPNav::TerrainSource::TERRAIN_UNAVAILABLE:
-                check_failed(ARMING_CHECK_PARAMETERS, display_failure, "RTL_ALT_TYPE=1 but no terrain data");
+                check_failed(ARMING_CHECK_PARAMETERS, display_failure, "RTL_ALT_TYPE=1, but no terrain data");
```
Comment (rmackay9) [inline ArduCopter/AP_Arming.cpp:240]: let's remove the comma here

File: ArduCopter/AP_Arming.cpp [hunk 3]
```diff
@@ -237,24 +237,24 @@ bool AP_Arming_Copter::parameter_checks(bool display_failure)
...
                 return false;
                 break;
             case AC_WPNav::TerrainSource::TERRAIN_FROM_RANGEFINDER:
                 if (!copter.rangefinder_state.enabled || !copter.rangefinder.has_orientation(ROTATION_PITCH_270)) {
-                    check_failed(ARMING_CHECK_PARAMETERS, display_failure, "RTL_ALT_TYPE=1 but no rangefinder");
+                    check_failed(ARMING_CHECK_PARAMETERS, display_failure, "RTL_ALT_TYPE=1, but no rangefinder");
```
Comment (rmackay9) [inline ArduCopter/AP_Arming.cpp:245]: let's remove the comma here

File: ArduCopter/AP_Arming.cpp [hunk 4]
```diff
@@ -237,24 +237,24 @@ bool AP_Arming_Copter::parameter_checks(bool display_failure)
...
                     return false;
                 }
                 // check if RTL_ALT is higher than rangefinder's max range
                 if (copter.g.rtl_altitude > copter.rangefinder.max_distance_cm_orient(ROTATION_PITCH_270)) {
-                    check_failed(ARMING_CHECK_PARAMETERS, display_failure, "RTL_ALT_TYPE=1 but RTL_ALT>RNGFND_MAX_CM");
+                    check_failed(ARMING_CHECK_PARAMETERS, display_failure, "RTL_ALT_TYPE=1, but RTL_ALT>RNGFND_MAX_CM");
```
Comment (rmackay9) [inline ArduCopter/AP_Arming.cpp:250]: let's remove the comma here

File: ArduCopter/AP_Arming.cpp [hunk 5]
```diff
@@ -237,24 +237,24 @@ bool AP_Arming_Copter::parameter_checks(bool display_failure)
...
                     return false;
                 }
                 break;
             case AC_WPNav::TerrainSource::TERRAIN_FROM_TERRAINDATABASE:
 #if AP_TERRAIN_AVAILABLE && AC_TERRAIN
                 if (!copter.terrain.enabled()) {
-                    check_failed(ARMING_CHECK_PARAMETERS, display_failure, "RTL_ALT_TYPE=1 but terrain disabled");
+                    check_failed(ARMING_CHECK_PARAMETERS, display_failure, "RTL_ALT_TYPE=1, but terrain disabled");
```
Comment (rmackay9) [inline ArduCopter/AP_Arming.cpp:257]: let's remove the comma here

**Comment by khancyr** [comment]
done !

**Comment by rmackay9** [comment]
Merged, thanks!

**Pull request merged**